static/src/js/actions/__tests__/mocks.js

```diff
@@ -1 +1 @@
-export const testJwtToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwidXNlcm5hbWUiOiJ0ZXN0dXNlciIsInByZWZlcmVuY2VzIjp7Im1hcFZpZXciOnsiem9vbSI6NCwibGF0aXR1ZGUiOjM5LCJiYXNlTGF5ZXIiOiJibHVlTWFyYmxlIiwibG9uZ2l0dWRlIjotOTUsInByb2plY3Rpb24iOiJlcHNnNDMyNiIsIm92ZXJsYXlMYXllcnMiOlsicmVmZXJlbmNlRmVhdHVyZXMiLCJyZWZlcmVuY2VMYWJlbHMiXX0sInBhbmVsU3RhdGUiOiJkZWZhdWx0IiwiZ3JhbnVsZVNvcnQiOiJkZWZhdWx0IiwiY29sbGVjdGlvblNvcnQiOiJkZWZhdWx0IiwiZ3JhbnVsZUxpc3RWaWV3IjoiZGVmYXVsdCIsImNvbGxlY3Rpb25MaXN0VmlldyI6ImRlZmF1bHQifSwidXJzUHJvZmlsZSI6eyJmaXJzdF9uYW1lIjoidGVzdCJ9LCJlYXJ0aGRhdGFFbnZpcm9ubWVudCI6InByb2QiLCJpYXQiOjE2NzM2NDY2NTR9.OU3QqvWSWNxUtU4xrZNMRBjZqGzXHoGW2E9IjOh1WuI'
+export const testJwtToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwidXNlcm5hbWUiOiJ0ZXN0dXNlciIsInByZWZlcmVuY2VzIjp7Im1hcFZpZXciOnsiem9vbSI6NCwibGF0aXR1ZGUiOjM5LCJiYXNlTGF5ZXIiOiJ3b3JsZEltYWdlcnkiLCJsb25naXR1ZGUiOi05NSwicHJvamVjdGlvbiI6ImVwc2c0MzI2Iiwib3ZlcmxheUxheWVycyI6WyJyZWZlcmVuY2VGZWF0dXJlcyIsInJlZmVyZW5jZUxhYmVscyJdfSwicGFuZWxTdGF0ZSI6ImRlZmF1bHQiLCJncmFudWxlU29ydCI6ImRlZmF1bHQiLCJjb2xsZWN0aW9uU29ydCI6ImRlZmF1bHQiLCJncmFudWxlTGlzdFZpZXciOiJkZWZhdWx0IiwiY29sbGVjdGlvbkxpc3RWaWV3IjoiZGVmYXVsdCJ9LCJ1cnNQcm9maWxlIjp7ImZpcnN0X25hbWUiOiJ0ZXN0In0sImVhcnRoZGF0YUVudmlyb25tZW50IjoicHJvZCIsImlhdCI6MTY3MzY0NjY1NH0.Z_qDm4aWpSRMxN4tpcyLJ-ZvZP-mLzFlzINMDR03VLA'
```

Comment:
The reason this gets updated is because if you go on https://jwt.io/ this has blueMarble in it so it needs to be updated